2.cellprofiling/scripts/area_shape_gpu.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
```

Comment:
Same comment, this code cell looks to not have been run but the rest of the notebook is fine.